Apps/GatewayApi/test/unit/Controllers.Test/UserProfileControllerTests.cs

```diff
@@ -579,7 +579,7 @@ private RequestResult<UserProfileModel> GetUserProfileExpectedRequestResultMock(
             userProfileServiceMock.Setup(s => s.GetUserPreferences(this.hdid))
                 .Returns(
                     new RequestResult<Dictionary<string, UserPreferenceModel>>
-                        { ResourcePayload = userPreferencePayloadMock });
```

Comment:
This does not match our formatting template. I think this should be discussed with team... @MikeLyttle

Reply:
fixed